sdk/textanalytics/ai-text-analytics/samples-dev/modelVersion.ts

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample shows how to choose model versions to use with pre-built Text
+ * Analytics models.
+ *
+ * @summary shows how to choose model versions for pre-built models.
+ * @azsdk-weight 40
+ */
+
+import { TextAnalyticsClient, AzureKeyCredential } from "@azure/ai-text-analytics";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+dotenv.config();
+
+// You will need to set these environment variables or edit the following values
+const endpoint = process.env["ENDPOINT"] || "<cognitive services endpoint>";
+const apiKey = process.env["TEXT_ANALYTICS_API_KEY"] || "<api key>";
+
+const documents = ["This document is written in English."];
+
+export async function main() {
+  console.log("== Choosing Model Version Sample ==");
+
+  const client = new TextAnalyticsClient(endpoint, new AzureKeyCredential(apiKey));
+
+  // TODO: show a list of different model versions to use from
+  const recognizeEntitiesResults = await client.recognizeEntities(documents, "en", {
+    /**
+     * Specify the model version by setting this property. "latest" indicates
+     * the latest generaly availability version of the model. When not specified,
+     * latest will be assumed. Model versions are date based, e.g "2021-06-01".
```

Comment:
maybe include the service doc that shows all the available modelVersions for each endpoint?

Reply:
Great point, I added a link to the list of versions for NER. I could not find the versions list for all features in one place. @aahill do you know where can I find such list?

Reply:
We are just about to publish one. It's a chicken-egg situation ;)